src/Features/Core/Portable/InitializeParameter/AbstractInitializeMemberFromParameterCodeRefactoringProviderMemberCreation.cs

```diff
@@ -190,6 +225,8 @@ private static string GenerateUniqueName(IParameterSymbol parameter, ImmutableAr
             throw ExceptionUtilities.Unreachable;
         }
 
+        protected abstract Accessibility DetermineDefaultPropertyAccessibility();
```

Comment:
small nit: can you put all the abstract members at the top of the file together?